### PR TITLE
#39 Create test case for PsChain class

### PR DESCRIPTION
**Pull request by eg04lt3r**

### CODE REVIEW

**Comment by davvd** [comment]
I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by davvd** [comment]
@ggajos it's yours, please review

**Comment by eg04lt3r** [comment]
I fixed build issues, please, try again.

**Comment by ggajos** [comment]
@eg04lt3r please fix qulice errors

**Comment by eg04lt3r** [comment]
Please, review code after check style fixes.

**Comment by ggajos** [comment]
@rultor merge please

File: src/test/java/org/takes/facets/auth/PsChainTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+/**
+ * Test case for {@link PsChain}.
+ * @author Aleksey Kurochka (eg04lt3r@gmail.com)
+ * @version $Id$
+ */
+public final class PsChainTest {
+
+    /**
+     * Check that PsChain returns proper identity.
```
Comment (yegor256) [inline src/test/java/org/takes/facets/auth/PsChainTest.java]: check this: http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names

File: src/test/java/org/takes/facets/auth/PsChainTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            new PsChain(
+                    new PsLogout(),
+                    new PsFake(true)
+            ).enter(new RqFake()).next(),
+            Matchers.is(Identity.ANONYMOUS)
+        );
+    }
+
+    /**
+     * Check that exit method of PsChain returns proper response.
```
Comment (yegor256) [inline src/test/java/org/takes/facets/auth/PsChainTest.java]: same here: http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names

File: src/test/java/org/takes/facets/auth/PsChainTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    /**
+     * Check that PsChain returns proper identity.
+     * @throws IOException if some problems inside
+     */
+    @Test
+    public void chainExecutionTest() throws IOException {
+        MatcherAssert.assertThat(
+            new PsChain(
+                    new PsLogout(),
```
Comment (yegor256) [inline src/test/java/org/takes/facets/auth/PsChainTest.java]: why 8 indentation spaces here?

**Comment by yegor256** [comment]
@eg04lt3r a few comments from me above

**Comment by eg04lt3r** [comment]
Thanks for comments. I resolved these issues in last commit.

2015-03-30 20:08 GMT+03:00 Yegor Bugayenko <notifications@github.com>:

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by eg04lt3r** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor merge it

**Comment by davvd** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor deploy

**Comment by elenavolokhova** [comment]
@eg04lt3r According to our [quality rules](http://www.teamed.io/qa.html): 
> Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

There is no description provided. Pull request must briefly explain what was done in the branch, so that we can use it for future reference. Please, fix it.

**Comment by elenavolokhova** [comment]
@ggajos According to our [quality rules](http://www.teamed.io/qa.html): 
> The code reviewer found at least three problems in the code. 

There were no issues found during review. Please, try to find at least three problems next time.

**Comment by yegor256** [comment]
@elenavolokhova :+1:

**Comment by elenavolokhova** [comment]
@davvd Quality of this one is acceptable.

**Comment by davvd** [comment]
> @davvd Quality of this one is acceptable.

@elenavolokhova thanks a lot

**Comment by davvd** [comment]
@ggajos I added 10 mins to @elenavolokhova (for QA review) in transaction 54991523... I added **17 mins** to your account, many thanks for working with the project! The completion time here was AP-8LM817457L067321J.... extra minutes for review comments (c=2)... +17 added to your rating, at the moment it is: [+3469](http://www.netbout.com/b/35094?open=rating)

**Pull request merged**